### PR TITLE
CallTraces prune to use ETL

### PR DESCRIPTION
**Pull request by AskAlexSharov**

### CODE REVIEW

**Comment by mikeshultz** [comment]
Tried testing this out, got a panic:

    Aug 28 20:48:30 eth erigon[2762]: [INFO] [08-28|20:48:30.910] Maximum peer count                       ETH=100 total=100
    Aug 28 20:48:30 eth erigon[2762]: [INFO] [08-28|20:48:30.910] Set global gas cap                       cap=50000000
    Aug 28 20:48:30 eth erigon[2762]: [INFO] [08-28|20:48:30.975] Opening Database                         label=chaindata path=/home/eth/.local/share/erigon/chaindata
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.108] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Byzantium: 4370000 Constantinople: 7280000 Petersburg: 7280000 Istanbul: 9069000, Muir Glacier: 9200000, Berlin: 12244000, London: 12965000, Engine: ethash}"
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.108] Disk storage enabled for ethash DAGs     dir=/home/eth/.local/share/erigon/ethash-dags count=2
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.108] Initialising Ethereum protocol           network=1
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.108] Effective                                prune="--prune=c"
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.108] Starting private RPC server              on=127.0.0.1:9090
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.922] [1/18 Headers] Waiting for headers...    from=13115973
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.975] Started P2P networking                   version=66 self=enode://NNNN@127.0.0.1:30303 name=erigon/v2021.09.1-alpha-b00936cd/linux-amd64/go1.17
    Aug 28 20:48:31 eth erigon[2762]: [INFO] [08-28|20:48:31.982] Started P2P networking                   version=65 self=enode://NNNN@127.0.0.1:30304 name=erigon/v2021.09.1-alpha-b00936cd/linux-amd64/go1.17
    Aug 28 20:49:00 eth erigon[2762]: [INFO] [08-28|20:49:00.773] Generating ethash verification cache     epoch=437 percentage=94 elapsed=3.008s
    Aug 28 20:49:00 eth erigon[2762]: [INFO] [08-28|20:49:00.950] Generated ethash verification cache      epoch=437 elapsed=3.185s
    Aug 28 20:49:01 eth erigon[2762]: [INFO] [08-28|20:49:01.923] [1/18 Headers] Inserting headers         progress=13116251
    Aug 28 20:49:03 eth erigon[2762]: [INFO] [08-28|20:49:03.952] Generating ethash verification cache     epoch=438 percentage=93 elapsed=3.000s
    Aug 28 20:49:04 eth erigon[2762]: [INFO] [08-28|20:49:04.153] Generated ethash verification cache      epoch=438 elapsed=3.201s
    Aug 28 20:49:28 eth erigon[2762]: [INFO] [08-28|20:49:28.108] [1/18 Headers] Processed                 highest inserted=13117798 age=36s
    Aug 28 20:49:29 eth erigon[2762]: [INFO] [08-28|20:49:29.099] [4/18 Bodies] Processing bodies...       from=13115973 to=13117798
    Aug 28 20:49:44 eth erigon[2762]: [INFO] [08-28|20:49:44.987] [4/18 Bodies] Processed                  highest=13117798
    Aug 28 20:49:44 eth erigon[2762]: [INFO] [08-28|20:49:44.987] [6/18 Senders] Started                   from=13115973 to=13117798
    Aug 28 20:49:59 eth erigon[2762]: [INFO] [08-28|20:49:59.054] [7/18 Execution] Blocks execution        from=13115973 to=13117798
    Aug 28 20:50:00 eth erigon[2762]: panic: 1
    Aug 28 20:50:00 eth erigon[2762]: goroutine 55 [running]:
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/eth/stagedsync.executeBlock(0xc0336f7ef0, {0x1e29b78, 0xc0079e2330}, {0x7d72f84f4c10, _}, {{0x1e076f8, 0xc000522000}, 0x20000000, {0x1, 0xffffffffffffffff, ...}, ...}, ...)
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/eth/stagedsync/stage_execute.go:132 +0x417
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/eth/stagedsync.SpawnExecuteBlocksStage(0xc0336d1c80, {0x1dde760, 0xc0002e7c30}, {0x0, 0x0}, 0x0, {0x1dff9f8, 0xc000b31500}, {{0x1e076f8, 0xc000522000}, ...}, ...)
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/eth/stagedsync/stage_execute.go:299 +0xa28
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/eth/stagedsync.DefaultStages.func19(0x30, 0x13bc8f0, {0x1dde760, 0xc0002e7c30}, {0x0, 0x0})
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/eth/stagedsync/default_stages.go:119 +0x8f
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/eth/stagedsync.(*Sync).runStage(0xc0002e7c30, 0xc00008a7d0, {0x1e076f8, 0xc000522000}, {0x0, 0x0}, 0x0)
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/eth/stagedsync/sync.go:312 +0x148
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/eth/stagedsync.(*Sync).Run(0xc0002e7c30, {0x1e076f8, 0xc000522000}, {0x0, 0x0}, 0x0)
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/eth/stagedsync/sync.go:229 +0x27c
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/turbo/stages.StageLoopStep({0x1dff9f8, 0xc000b31500}, {0x1e076f8, 0xc000522000}, 0xc0002e7c30, 0x0, 0xc00018c480, 0x1, 0xc00112ca50, 0x0)
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/turbo/stages/stageloop.go:127 +0x42b
    Aug 28 20:50:00 eth erigon[2762]: github.com/ledgerwatch/erigon/turbo/stages.StageLoop({0x1dff9f8, 0xc000b31500}, {0x1e076f8, 0xc000522000}, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/turbo/stages/stageloop.go:54 +0x15f
    Aug 28 20:50:00 eth erigon[2762]: created by github.com/ledgerwatch/erigon/eth.(*Ethereum).Start
    Aug 28 20:50:00 eth erigon[2762]:  github.com/ledgerwatch/erigon/eth/backend.go:687 +0x3d8
    Aug 28 20:50:00 eth systemd[1]: erigon.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
    Aug 28 20:50:00 eth systemd[1]: erigon.service: Failed with result 'exit-code'.
    Aug 28 20:50:00 eth systemd[1]: erigon.service: Consumed 1min 10.789s CPU time.

File: eth/stagedsync/stage_execute.go [hunk 1]
```diff
@@ -128,7 +128,9 @@ func executeBlock(
 			cfg.changeSetHook(blockNum, hasChangeSet.ChangeSetWriter())
 		}
 	}
-
+	if writeCallTraces {
+		panic(1)
```
Comment (mikeshultz) [inline eth/stagedsync/stage_execute.go:132]: Leftover debugging?

**Comment by AskAlexSharov** [comment]
@mikeshultz hehe, this was hardcoded panic for my own test. Removed.

**Comment by mikeshultz** [comment]
Tried out this branch and was able to sync overnight.  :+1:

**Pull request merged**